spec/models/manageiq/providers/redhat/infra_manager/provision/state_machine_spec_helper.rb

```diff
@@ -0,0 +1,65 @@
+module ManageIQ::Providers::Redhat::InfraManager::Provision::StateMachineSpecHelper
+  include MiqProvision::StateMachineSpecHelper
```

Comment:
This file provides test methods for the methods in `ManageIQ::Providers::Redhat::InfraManager::Provision::StateMachine`, `MiqProvision::StateMachineSpecHelper` provides test methods for methods defined in `MiqProvision::StateMachine`